package.json

```diff
@@ -268,7 +268,7 @@
     "@department-of-veterans-affairs/component-library": "11.2.6",
     "@department-of-veterans-affairs/formation": "^7.0.2",
     "@department-of-veterans-affairs/react-jsonschema-form": "^1.2.5",
-    "@department-of-veterans-affairs/va-forms-system-core": "1.2.2",
+    "@department-of-veterans-affairs/va-forms-system-core": "~1.3.0",
```

Comment:
This will allow for any `patches` to be picked up automatically up to version `1.4.0`. Do we want to do that? What is the standard for vets-website? @micahchiang

Reply:
@bjmccotter7192 @micahchiang if we're patching correctly, it shouldn't be a problem, but I don't want to jinx it by saying that